components/TextControls/BUILD

```diff
@@ -43,29 +43,29 @@ package_group(
 
 mdc_public_objc_library(
```

Comment:
This is the often the wild card component that depends on all of the others. I was expecting to see some deps for this one.

Reply:
Randall and I to discuss offline, but we should not be creating wildcard umbrella targets for our components. Clients are expected to pick the specific components they intend to use.

Reply:
Initially when I opened this PR this TextControls target only included the shared public types, it wasn't an umbrella target that depended on/included everything. After my changes yesterday, the TextControls target now exists as TextControls:shared.